### PR TITLE
Add nodejs auto-instrumentation for runtime metrics in paymentservice

### PR DESCRIPTION
**Pull request by liyanwei93**
# Changes

This adds auto-instrumentation for nodejs runtime metrics in paymentservice.  Users can automatically get some runtime metrics to enrich metrics monitoring.

## Merge Requirements

For new features contributions, please make sure you have completed the following
essential items:

* [x] `CHANGELOG.md` updated to document new feature additions
* [ ] Appropriate documentation updates in the [docs][]
* [ ] Appropriate Helm chart updates in the [helm-charts][]

Maintainers will not merge until the above have been completed. If you're unsure
which docs need to be changed ping the
[@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers).

[docs]: https://opentelemetry.io/docs/demo/
[helm-charts]: https://github.com/open-telemetry/opentelemetry-helm-charts

### CODE REVIEW

**Comment by julianocosta89** [comment]
Hey @liyanwei93 this is actually a nice PR.
Would you be able to sign the EasyCLA and re-open it?

**Comment by liyanwei93** [comment]
Hi @julianocosta89, thanks for the reply! Because our company (IBM) has not yet signed a CCLA, I am applying for the right person to be the CLA manager for our company. I will continue this PR once I sign the EasyCLA. cc @MorningSpace

**Comment by julianocosta89** [comment]
@liyanwei93 I've converted it to draft.
Whenever you get the approval to sign the EasyCLA, we can move to ready to review 🤩

**Comment by liyanwei93** [comment]
Hi @julianocosta89, this PR is ready to review, could you please help? Thanks

File: CHANGELOG.md
Comment (julianocosta89) [inline CHANGELOG.md:26]:
```suggestion
* [paymentservice] Add nodejs instrumentation for runtime metrics
```

**Review by julianocosta89 — APPROVED** [review]
Just a minor note on the changelog, but LGTM

File: CHANGELOG.md
Comment (liyanwei93) [reply CHANGELOG.md]: I have updated it. Thanks!!

**Review by julianocosta89 — APPROVED** [review]

**Comment by liyanwei93** [comment]
Hi @julianocosta89 Sorry to bother you again, May I know how to get workflow running?

**Comment by julianocosta89** [comment]
@liyanwei93 as this is your first contribution to the repo, the workflow needs to be manually triggered by one of the approvers/maintainers.

Could you update you branch?
This should be good to be merged

**Comment by liyanwei93** [comment]
Hi @julianocosta89, I have updated my branch, can you help to trigger the workflow again and merge it? Thanks!

**Pull request merged**
